visidata/vd.py

```diff
@@ -1471,6 +1471,9 @@ def read_bytes(self):
     def is_dir(self):
         return os.path.isdir(self.resolve())
 
+    def exists(self):
+        return os.path.exists(self.resolve())
```

Comment:
this might not be the desired function in some cases, see [here](http://stackoverflow.com/questions/82831/how-do-i-check-whether-a-file-exists-using-python). But for the example, where we know that it *will* be a file, it should be good enough?